examples/stable_diffusion_v2/tools/eval/eval_clip_i_score.py

```diff
@@ -64,18 +64,18 @@ def load_images(image_path_or_dir):
         type=str,
         help="backend to do CLIP model inference for CLIP score compute. Option: ms, pt." " Default: ms",
     )
-    parser.add_argument("--load_checkpoint", default=None, type=str, help="load model checkpoint." " Default: None")
+    parser.add_argument("--ckpt_path", default=None, type=str, help="load model checkpoint." " Default: None")
     parser.add_argument(
         "--tokenizer_path",
-        default="bpe_simple_vocab_16e6.txt.gz",
+        default="ldm/models/clip/bpe_simple_vocab_16e6.txt.gz",
         type=str,
-        help="load model checkpoint." " Default: bpe_simple_vocab_16e6.txt.gz",
+        help="load model checkpoint." " Default: ldm/models/clip/bpe_simple_vocab_16e6.txt.gz",
     )
     parser.add_argument("--quiet", action="store_true", help="set this flag to avoid printing scores")
     parser.add_argument(
-        "--no_check_certificate",
+        "--check_certificate",
```

Comment:
check_certificate is a too trivial arg, we can leave it to False always in this case.

Reply:
ok